apps/agent/package.json

```diff
@@ -35,23 +32,25 @@
   "license": "(GPL-3.0-or-later OR AGPL-3.0-or-later)",
   "description": "",
   "devDependencies": {
-    "@aragon/apps-shared-migrations": "1.0.0",
-    "@aragon/apps-shared-scripts": "^1.0.0",
     "@aragon/cli": "^6.0.0",
-    "@aragon/test-helpers": "^2.1.0",
-    "@aragon/truffle-config-v4": "^1.0.1",
+    "@aragon/contract-test-helpers": "0.0.1",
     "eth-ens-namehash": "^2.0.8",
     "eth-gas-reporter": "^0.2.0",
     "ethereumjs-testrpc-sc": "^6.5.1-sc.0",
     "ethereumjs-util": "^6.1.0",
     "ganache-cli": "^6.4.3",
     "solidity-coverage": "0.6.2",
     "solium": "^1.2.3",
-    "truffle": "4.1.14",
-    "truffle-extract": "^1.2.1",
-    "web3-eth-abi": "^1.2.1"
+    "web3-eth-abi": "^1.2.1",
+    "@aragon/buidler-aragon": "^0.1.0",
+    "@nomiclabs/buidler": "^1.0.2",
+    "@nomiclabs/buidler-truffle5": "^1.1.2",
+    "@nomiclabs/buidler-web3": "^1.1.2",
+    "bignumber.js": "^9.0.0",
+    "web3": "^1.2.6"
```

Comment:
Ah, let's re-order these dependencies; npm should automatically do so if you add a new package (just remove it afterwards).